src/components/Tabs/TabList/TabList.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../styles/themes";
+@import "../../../styles/typography";
+
+.tabsWrapper {
+  padding-bottom: 20px;
+}
+
+.tabsList {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  font-size: 0;
+  position: relative;
+  height: 40px;
+  outline: none;
+}
+
+.tabsList.sm {
+  height: 32px;
+}
+
+.tabsList.lg {
+  height: 48px;
+}
+
+.stretched .tabsList {
+  width: 100%;
+}
+
+.stretched .tabsList :global(.tab--wrapper) {
```

Comment:
global is for using css modules global classes right? so why we are using a class name which not belong to css modules? how does it work? (tab--wrapper is with bem convetion which we do not use in css modules)